config/routes.rb

```diff
@@ -75,6 +75,7 @@
     resources :ldap_test
     resources :dex_connector_ldaps, path: :ldap_connectors
     resources :dex_connector_oidcs, path: :oidc_connectors
+    resources :ext_cert, only: [:index, :create]
```

Comment:
ext_cert looks kind of ugly. Would be great to changed it everywhere to 'external_cert'. This is a personal preference thing, as people will agree/disagree on whether 'ext' clearly indicates 'external' versus 'cert' clearly being 'certificate'. I think it is worth changing for clarity. 'cert' is fine in my mind because it is very obvious. 'ext' could also be 'extend'

Reply:
Sounds good to me.  Done.